public/tutorials/robust-smart-contracts-with-openzeppelin.md

```diff
@@ -112,51 +114,102 @@ Using less than 15 lines of hand-coded Solidity, we've created our own Ethereum
 
    The `import` statement within our `TutorialToken` contract will be automatically handled by the compiler, along with any subsequent imports within `StandardToken`.
 
-1. Now we can compile and migrate. In a second terminal window, run the TestRPC:
+1. Now we are ready to compile and migrate to the blockchain. We will do this using Truffle Develop, a development console that includes an in-memory blockchain, similar to the [EthereumJS TestRPC](https://github.com/ethereumjs/testrpc). In your terminal, make sure you are in the root of your project's directory and then launch Truffle Develop:
 
    ```shell
-   testrpc
+   truffle develop
    ```
 
-1. Back in the first terminal window, run the following commands to compile and migrate the contract to the TestRPC:
+   <p class="alert alert-info">
+     <strong>Note</strong>: If you're on Windows and encountering problems running this command, please see the documentation on [resolving naming conflicts on Windows](/docs/advanced/configuration#resolving-naming-conflicts-on-windows).
+   </p>
+
+1. Once the Truffle Develop prompt displays, run the following command to compile the contract:
 
    ```shell
-   truffle compile
-   truffle migrate
+   compile
+   ```
+
+1. Once the compile has completed, migrate the contract to the blockchain:
+
+   ```shell
+   migrate
    ```
 
    <p class="alert alert-info">
-   <strong>Note</strong>: If you're on Windows and encountering problems running these commands, please see the documentation on [resolving naming conflicts on Windows](/docs/advanced/configuration#resolving-naming-conflicts-on-windows).
+     <strong>Note</strong>: As an alternative to using Truffle Develop, both the `compile` and `migrate` commands can be run directly on the command line with `truffle compile` and `truffle migrate` respectively.
```

Comment:
This sentence is misleading (I believe; @gnidan, correct me if I'm wrong here). `truffle migrate` won't use the `truffle develop` blockchain unless you have `truffle develop` already running and the command running inside the console provided by `truffle develop`. `truffle compile` doesn't need a blockchain environment as it doesn't talk to the blockchain, but `truffle migrate` does, and the only way it knows about the `develop` blockchain is if its within a `truffle develop` console.

Reply:
This is all correct – `truffle migrate` has no way to use the Truffle Develop blockchain, unless invoked as `migrate` from within the Develop console

Reply:
Right, of course, I neglected to mention there that you would have to use those commands *while connected to some other blockchain* (such as testrpc). Didn't mean to imply that Truffle Develop would know about it. Will update.